tests/garage/torch/modules/test_gaussian_mlp_module.py

```diff
@@ -0,0 +1,362 @@
+import pickle
+
+import pytest
+import torch
+from torch import nn
+
+from garage.torch.modules.gaussian_mlp_module import GaussianMLPModule
+
+plain_settings = [
+    (5, 1, (1,)),
+    (5, 1, (2,)),
+    (5, 2, (3,)),
+    (5, 2, (1, 1)),
+    (5, 3, (2, 2))
+]
+
+
+different_std_settings = [
+    (5, 1, (1,), (1,)),
+    (5, 1, (2,), (2,)),
+    (5, 2, (3,), (3,)),
+    (5, 2, (1, 1), (1, 1)),
+    (5, 3, (2, 2), (2, 2))
+]
+
+
+@pytest.mark.parametrize("input_dim, output_dim, hidden_sizes", plain_settings)
+def test_std_share_network_output_values(input_dim, output_dim, hidden_sizes):
+    module = GaussianMLPModule(
+        input_dim=input_dim,
+        output_dim=output_dim,
+        hidden_sizes=hidden_sizes,
+        std_share_network=True,
+        hidden_nonlinearity=None,
+        std_parameterization='exp',
+        hidden_w_init=nn.init.ones_,
+        output_w_init=nn.init.ones_
+    )
+
+    dist = module(torch.ones(5))
+
+    exp_mean = torch.full((output_dim, ), 5 * (torch.Tensor(hidden_sizes).prod().item()))
```

Comment:
shouldn't this be np.full, etc? the mean and variance are numpy types.

Reply:
the mean and variance from the distribution are torch.tensor type.
I think mean and variance from the GaussianMLPPolicy was numpy type.(currently removed)
So, I compare the mean and variance with torch.tensor type expected value